packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -398,13 +477,51 @@ function WorklistDataGrid(props) {
     }
   };
 
-  const handleCellSelection = (cellRowIdColId, cellCoordinates) => {
-    selectCell(cellRowIdColId, cellCoordinates);
+  const handleCellSelection = (selectionDetails) => {
+    selectCell(selectionDetails);
   };
 
-  const handleRowSelection = (rowId, rowIndex, selectedCellCoordinates) => {
-    handleCellSelectionChange(null, null, selectedCellCoordinates);
-    selectRows(false, rowId, rowIndex);
+  const handleRowSelection = (selectionDetails) => {
+    handleCellSelectionChange(null, null, { row: selectionDetails.rowIndex, col: selectionDetails.columnIndex });
+    const selectionData = selectionDetails.selectedByKeyboard ? rowSelection.current.lastIndividuallySelectedRow : rowSelection?.current?.focussedRow;
```

Comment:
This variable name doesn't really seem to clearly describe the purpose of the variable.

Reply:
Updated.